src/Simplex/Chat/Store.hs

```diff
@@ -336,6 +337,22 @@ getUserContacts st User {userId} =
     contactNames <- liftIO $ map fromOnly <$> DB.query db "SELECT local_display_name FROM contacts WHERE user_id = ?" (Only userId)
     rights <$> mapM (runExceptT . getContact_ db userId) contactNames
 
+getPendingConnections :: MonadUnliftIO m => SQLiteStore -> User -> m [Connection]
+getPendingConnections st User {userId} =
+  liftIO . withTransaction st $ \db ->
+    map toConnection
+      <$> DB.queryNamed
+        db
+        [sql|
+          SELECT c.connection_id, c.agent_conn_id, c.conn_level, c.via_contact,
+            c.conn_status, c.conn_type, c.contact_id, c.group_member_id, c.created_at
+          FROM connections c
+          WHERE c.user_id = :user_id
+            AND c.conn_type = 'contact'
```

Comment:
better to pass it as a parameter (ConnContact), not to use the mapping in two places